completions/bash/kpod

```diff
@@ -664,6 +664,33 @@ _kpod_attach() {
      _complete_ "$options_with_args" "$boolean_options"
 }
 
+_kpod_commit() {
```

Comment:
You need a commit added to the section below.

Reply:
fixed